src/emqx_acl_cache.erl

```diff
@@ -48,7 +49,7 @@
 %% Wrappers for key and value
 cache_k(PubSub, Topic)-> {PubSub, Topic}.
 cache_v(AclResult)-> {AclResult, time_now()}.
-discard_k() -> {?MODULE, discard_stamp}.
+drain_k() -> {?MODULE, discard_stamp}.
```

Comment:
Nit: Renaming the key `discard_stamp -> drain_stamp` will make it more greppable in the code